packages/destination-actions/src/destinations/sendgrid/sendgrid-properties.ts

```diff
@@ -0,0 +1,22 @@
+import { InputField } from '@segment/actions-core'
+import { Payload } from './updateUserProfile/generated-types'
+
+export const customFields: InputField = {
+  label: 'Other Fields',
+  description: `
+  Additional fields to send to SendGrid. On the left-hand side, input the SendGrid Custom Fields Id. On the right-hand side, map the Segment field that contains the value.
+
+  Custom Fields must be predefined in your SendGrid account and you can retrieve corresponding Id using get all field definitions endpoint.
+
+  Reference: https://docs.sendgrid.com/api-reference/custom-fields/get-all-field-definitions
```

Comment:
I think we can use markdown to hyperlink this instead. @dlasky or @LLuque-twilio can you confirm?

Reply:
Yep! Markdown is fully supported for description fields

Reply:
Nice! Found an example here: https://github.com/segmentio/action-destinations/blob/main/packages/destination-actions/src/destinations/salesforce/sf-properties.ts#L20

Reply:
```suggestion
  See more information on custom fields in the [SendGrid API documentation](https://docs.sendgrid.com/api-reference/custom-fields/get-all-field-definitions).
```
Something like this?